scripts/spellcheck.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+results=./Spellchecker.results
+> $results
```

Comment:
???

Reply:
It's for emptying the file. I know it's of no use in the action to run the spellcheck since the file will be new and always empty, but when I'm spellchecking on local, I don't want my results to be appended at the end of the previous results.

Given how the script works, this is the best and most simple way of ensuring the results file is clean before each execution.

Reply:
To be honest I didn't even know you could just `> $something`  to empty it, I looked it up